feature-authentication/src/main/java/com/apk/herbiary/screens/login/ui/GoogleSignInButton.kt

```diff
@@ -0,0 +1,42 @@
+package com.apk.herbiary.screens.login.ui
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.apk.herbiary.feature.authentication.R
+
+@Composable
+fun GoogleSignInButton(onClick: () -> Unit){
```

Comment:
Maybe instead of using multiple button components we can make only one with all the stuff we need, we can check this later